collector/indices_settings.go

```diff
@@ -90,6 +90,21 @@ func NewIndicesSettings(logger log.Logger, client *http.Client, url *url.URL) *I
 					return val
 				},
 			},
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "indices_settings", "number_of_replicas"),
```

Comment:
"number_of" is redundant. We can simplify this.

```suggestion
					prometheus.BuildFQName(namespace, "indices_settings", "replicas"),
```